src/main.cpp

```diff
@@ -86,6 +93,37 @@ void onExitSignal(int signum) {
 }
 
 
+bool setLockFile(QString configPath) {
+
+    QDir configDir(configPath);
+
+    if (! configDir.mkpath(".")) {
+        qFatal("Could not create config file.");
```

Comment:
Directory, not file
Also, don't end single sentences with a period, to match the style used by the UI and other log lines (applies to other places where you logged too).